sbin/build.sh

```diff
@@ -263,22 +263,33 @@ removingUnnecessaryFiles()
 signRelease()
 {
   if [ "$SIGN" ]; then
-    if [[ "$OSTYPE" == "cygwin" ]]; then
-      echo "Signing release"
-      signToolPath=${signToolPath:-"/cygdrive/c/Program Files/Microsoft SDKs/Windows/v7.1/Bin/signtool.exe"}
-      # Sign .exe files
-      FILES=$(find "${OPENJDK_REPO_TAG}" -type f -name '*.exe')
-      for f in $FILES; do
-        "$signToolPath" sign /f "$CERTIFICATE" /p "$SIGN_PASSWORD" /fd SHA256 /t http://timestamp.verisign.com/scripts/timstamp.dll "$f"
-      done
-      # Sign .dll files
-      FILES=$(find "${OPENJDK_REPO_TAG}" -type f -name '*.dll')
-      for f in $FILES; do
-        "$signToolPath" sign /f "$CERTIFICATE" /p "$SIGN_PASSWORD" /fd SHA256 /t http://timestamp.verisign.com/scripts/timstamp.dll "$f"
-      done
-    else
-      echo "Skiping code signing as it's only supported on Windows"
-    fi
+    case "$OSTYPE" in
+      "cygwin")
+        echo "Signing Windows release"
+        signToolPath=${signToolPath:-"/cygdrive/c/Program Files/Microsoft SDKs/Windows/v7.1/Bin/signtool.exe"}
+        # Sign .exe files
+        FILES=$(find "${OPENJDK_REPO_TAG}" -type f -name '*.exe')
+        for f in $FILES; do
+          "$signToolPath" sign /f "$CERTIFICATE" /p "$SIGN_PASSWORD" /fd SHA256 /t http://timestamp.verisign.com/scripts/timstamp.dll "$f"
+        done
+        # Sign .dll files
+        FILES=$(find "${OPENJDK_REPO_TAG}" -type f -name '*.dll')
+        for f in $FILES; do
+          "$signToolPath" sign /f "$CERTIFICATE" /p "$SIGN_PASSWORD" /fd SHA256 /t http://timestamp.verisign.com/scripts/timstamp.dll "$f"
+        done
+      ;;
+      "darwin"*)
+        echo "Signing OSX release"
+        # Sign all executables
+        FILES=$(find "${OPENJDK_REPO_TAG}" -perm +111 -type f)
+        for f in $FILES; do
```

Comment:
I don't think this handles spaces in paths (`./foo bar` becomes `./foo` and then `bar`), you need:

```bash
for f in "$FILES"; do
```

Reply:
if you do `for f in "$FILES"; do` then it will only loop once as it treats it as a string. I think that we're lucky enough that there aren't any spaces in any of the paths

Reply:
Whoops, I could have sworn that worked yesterday. Okay, POSIX way of doing it is:

```bash
echo "$FILES" | while read f; do codesign -s "$CERTIFICATE" "$f"; done
```